README.md

```diff
@@ -1,7 +1,13 @@
 # PuppetDB
 
-[docs]: https://docs.puppetlabs.com/puppetdb/latest
+[docs]: https://docs.puppet.com/puppetdb/latest
 
 PuppetDB is the fast, scalable, and reliable data warehouse for Puppet. It caches data generated by Puppet, and gives you advanced features at awesome speed with a powerful API.
 
 For documentation on this product, consult the [latest documentation][docs].
+
+## Maintenance
+
+Maintainers: Ryan Senior <ryan.senior@puppet.com>, Wyatt Alt <wyatt@puppet.com>, Rob Browning <rlb@puppet.com>, AJ Roetker <andrew.roetker@puppet.com>, Russell Mull <russell.mull@puppet.com>, Ken Barber <ken@puppet.com>
```

Comment:
I think there was some debate about putting aliases in an internal email thread at the moment, we can wait to see where that goes or chime in I think ... also, I'm not sure how public-ready the alias is, we might need to ensure we get moderation for example (as a moderator of puppet-users, this becomes a thing eventually).